src/types/sealedClass/sealedClass.kts

```diff
@@ -0,0 +1,16 @@
+package types.sealedClass
```

Comment:
Sealed class looks like has to have nested sub-classes when writing kotlin script (kts!)